en-US/Grammar.xml

```diff
@@ -146,6 +146,76 @@
 		</table>
 
 	</section>
+	<section id="possessives">
+	<title>Possessives</title>
+	<para>
+	A <emphasis>possessive</emphasis> indicates that something or someone belongs to a person or thing.
+	<!-- To form a possessive generally, for a singular noun add an apostrophe and <emphasis>s</emphasis>; for a plural noun, add an apostrophe after the final letter <emphasis>s</emphasis>. -->
+	</para>
+	 <para>
+	For guidance on how to form a possessive, see <xref linkend="apostrophes"/>.
+	</para>
+	<para>
+	Consider the following scenarios for when to use or not to use a possessive:
+	</para>
+	<para> 
+	Do not use possessives to refer to abbreviations or product names.
```

Comment:
For clarity, I suggest splitting out this guidance with their respective examples. From an information design standpoint, combining guidance can make it harder for readers to find the information they need. For example, I suggest starting with "Do not use possessives with product names." followed by the examples with product names. Then, "Do not use possessives with abbreviations." followed by the examples with abbreviations.

Reply:
Can you give me an example @rclee33? I had trouble with formatting.